sentinel-core/src/main/java/com/alibaba/csp/sentinel/util/SpiLoader.java

```diff
@@ -219,16 +233,16 @@
                 int order = SpiOrderResolver.resolveOrder(spi);
                 // Since SPI is lazy initialized in ServiceLoader, we use online sort algorithm here.
                 SpiOrderResolver.insertSorted(orderWrappers, spi, order);
-                RecordLog.info("[SpiLoader] Found {0} SPI: {1} with order " + order, clazz.getSimpleName(),
-                        spi.getClass().getCanonicalName());
+                RecordLog.debug("[SpiLoader] Found {} SPI: {} with order {}", clazz.getSimpleName(),
+                        spi.getClass().getCanonicalName(), order);
             }
             List<T> list = new ArrayList<>();
             for (int i = 0; i < orderWrappers.size(); i++) {
                 list.add(i, orderWrappers.get(i).spi);
```

Comment:
Although these codes are not changed in this PR but i should point out there are two ways to optimize a little, just FYI:

1. `list` should have an initial capacity specified because we know it(Same to `orderWrappers`)
2. It's an in order copying so `add(T obj)` is enough and will make sense.

Reply:
Good idea! It has been improved, please check.